ui/app/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -31,11 +31,15 @@ export default class AdvancedTab extends PureComponent {
     threeBoxSyncingAllowed: PropTypes.bool.isRequired,
     setThreeBoxSyncingPermission: PropTypes.func.isRequired,
     threeBoxDisabled: PropTypes.bool.isRequired,
+    setIpfsGateway: PropTypes.func.isRequired,
+    ipfsGateway: PropTypes.string.isRequired,
   }
 
   state = {
     autoLogoutTimeLimit: this.props.autoLogoutTimeLimit,
     logoutTimeError: '',
+    ipfsGateway: `https://${this.props.ipfsGateway}`,
```

Comment:
How does this handle a gateway a user enters that includes the `https://` prefix?

Reply:
It does not, pushing up a fix.

Reply:
The `https` was only appended for the purpose of checking if it was a valid URL. This is now done invisibly in the change and save handlers.